### DISCUSSION TITLE
Prioritize qutebrowser keybings over web page keybindings.

### DISCUSSION THREAD

**Discussion by ghost** [qutebrowser/qutebrowser#7754] [Q&A] [answered]
I'm new on qutebrowser, on youtube pressing j or k increases/decreases the volume, which is annoying. I guess jk are interpreted as the up/down keys. is there an option to make them only work in qutebrowser? i mean just scroll the page but not activate anything else. if this can be done for all keys or there is a better way to achieve it, let me know pls. thx

**Comment by The-Compiler** [comment, marked as answer]
This is already the case - qutebrowser keybindings always win over website keybindings. However, j/k is indeed bound to `scroll down` and `scroll up`, which emulates cursor keypresses (which has better scrolling behavior than using the Qt API).

You can bind them (or alternative keys) to use `:scroll-px` instead - though last time I tried, that works worse when there are multiple scrollable elements on a webpage.
